<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInterface</class>
 <widget class="QWidget" name="TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RailStation</string>
  </property>
  <widget class="QPushButton" name="acceptWindowButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="windowButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть кассу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queueButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>217</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть Очередь</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWindow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string comment="выберите окно"/>
   </property>
   <property name="placeholderText">
    <string>выберите окно</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinWindowCountBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinQueueCountBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество окон</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальный
размер очереди</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>361</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertLayout"/>
  </widget>
  <widget class="QSpinBox" name="spinTimerBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Частота появление пассажира
 (в секундах)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="timerButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="windowAllButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть все окна</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinServiceTimeBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
  </widget>
  <widget class="QPushButton" name="setServiceTimeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Время обслуживания
очереди(в секундах)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обслуженно:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="countPassLabel">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Состояние кассы:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 97, 97);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Закрыто</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
